pkg/utils/clusterinfo/tidb.go

```diff
@@ -0,0 +1,26 @@
+package clusterinfo
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type TiDB struct {
+	Common
+	StatusPort string `json:"status_port"`
+}
+
+func (t *TiDB) UnmarshalJSON(d []byte) error {
```

Comment:
Why do you need to write UnmarshalJSON specifically for TiDB?

Reply:
TiDB's `status_port` is a uint, and I want to return a uint. Maybe I can try to change they all into uint.